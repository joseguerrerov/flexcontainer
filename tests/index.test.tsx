
import React from 'react';
import { shallow } from 'enzyme';
import FlexContainer from '../src';

describe('FlexContainer Component', () => {
  it('Renders component', () => {
    const link = shallow(<FlexContainer />);
    expect(link).toMatchSnapshot();
  });
});
