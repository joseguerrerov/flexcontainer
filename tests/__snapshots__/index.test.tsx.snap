// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FlexContainer Component Render renders with props 1`] = `
<div
  style={
    Object {
      "alignContent": "center",
      "alignItems": "center",
      "alignSelf": "flex-end",
      "display": "flex",
      "flex": 1,
      "flexDirection": "column",
      "flexFlow": "column",
      "flexWrap": "nowrap",
      "justifyContent": "center",
      "order": 1,
    }
  }
>
  <p>
    Test
  </p>
</div>
`;

exports[`FlexContainer Component Render renders without props 1`] = `
<div
  style={
    Object {
      "display": "flex",
    }
  }
/>
`;
